### ISSUE TITLE
Onboarding tour for dashboard feature

### ISSUE THREAD

**Issue by KerstinFaye** [visualize-admin/visualization-tool#1589]
Tooltips that explain the dashboards functionalities
- different layout options (tav layout, dashboard, single url)
- dashboard layout: tall, horizontal, free canvas
- resize charts, reorder charts
- shared filtes
- edit chart

![Image](https://github.com/visualize-admin/visualization-tool/assets/127507692/d50479e7-f84d-4bc9-8fe4-fe4479d8dde5)

**Comment by sosiology** [comment]
Thanks for creating the issue @KerstinFaye. I really like how IOM has implemented this, but think we should maybe first start with a pop over to keep things simple if we include it in this phase of refinemnts - maybe something similar to how we have it for the merging of cubes. 

![Image](https://github.com/visualize-admin/visualization-tool/assets/29289679/8e6b3186-62e1-4c82-8aae-d7c9a1c31ab8)


Thoughts @ptbrowne @bprusinowski @adintegra?

**Comment by bprusinowski** [comment]
I agree with everything you said @sosiology – for the first step it would be easier to re-use the logic we already have for the merging of cubes 👨‍💻
